src/main/java/com/thebois/controllers/InventoryController.java

```diff
@@ -13,24 +13,23 @@
 public class InventoryController {
 
     private final InventoryView inventoryView;
-    private final IInventory colony;
+    private final IInventory inventory;
     private Map<ItemType, Integer> itemCounts;
 
     /**
      * Instantiates the controller with the view and a colony reference.
      *
      * @param inventoryView The view to be updated.
-     * @param colony              The colony to get information from, regarding the inventory.
+     * @param inventory     The colony to get information from, regarding the inventory.
```

Comment:
I hate to be that guy, but the javadoc itself should probably no longer mention the colony, right? :P